src/pages/workspaces/workspace/Workflows.js

```diff
@@ -106,6 +106,7 @@ const WorkflowCard = memoWithName('WorkflowCard', ({ listView, name, namespace,
     methodRepoMethod: { sourceRepo, methodVersion },
   } = config;
   const sourceRepoName = sourceRepo === 'agora' ? 'Terra' : Utils.normalizeLabel(sourceRepo);
+  const props = WorkspaceUtils.getWorkspaceEditControlProps(workspace);
```

Comment:
Suggest a more specific name for this (`workspaceEditControlProps` or similar). I would assume that `props` contained all the props passed into the component.

Reply:
I agree. `props` would only be appropriate if the scope was tiny.